src/test/mochitest/browser_dbg-preview-module.js

```diff
@@ -0,0 +1,27 @@
+/* Any copyright is dedicated to the Public Domain.
+ * http://creativecommons.org/publicdomain/zero/1.0/ */
+
+// Test hovering in a script that is paused on load
+// and doesn't have functions.
+add_task(async function() {
+  const dbg = await initDebugger("doc-scripts.html");
+  const { selectors: { getSelectedSource }, getState } = dbg;
+
+  navigate(dbg, "doc-on-load.html")
+
+  await waitForPaused(dbg);
+
+  const popupPreviewed = waitForDispatch(dbg, "SET_PREVIEW");
+  hoverAtPos(dbg, { line: 1, ch: 6 });
+  await popupPreviewed;
+  await assertPopup(dbg, { field: "foo", value: "1", expression: "obj" });
+  await assertPopup(dbg, { field: "bar", value: "2", expression: "obj" });
+
+  // hover over an empty position
```

Comment:
I suspect this is done to hide the tooltip ? Would you mind adding a comment or an info call to explain that ?